deps/rabbitmq_stomp/src/rabbit_stomp_processor.erl

```diff
@@ -140,21 +164,32 @@ initial_state(Configuration,
        %%       to override this value?
```

Comment:
Delete the adapter info. It's not needed anymore.

Reply:
The management UI shows that there is 1 AMQP 0.9.1 channel for the STOMP connection, which is wrong.